helm-charts/support/values.yaml

```diff
@@ -297,7 +297,7 @@ grafana:
   # prometheus and grafana.
   #
   # Grafana's memory use seems to increase over time but seems reasonable to
-  # stay below 200Mi for years to come. Grafana's CPU use seems miniscule with
+  # stay below 200Mi for years to come. Grafana's CPU use seems minuscule with
```

Comment:
e.g. here is a case of a "rare" word -- AFAIK `minuscule` is a more proper/older version, but recently `miniscule` also got to be used

Reply:
right. this is also why i like this being pre-commit, because it just fixes it and there's no discussion that needs to be had. pre-commit imperialism ftw ;)

Reply:
just to make sure it was not missed since I failed to mention in PR description -- there is pre-commit config provided too in this PR: https://github.com/2i2c-org/infrastructure/pull/3098/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9